docs/chainlink-nodes/oracle-jobs/task-types/task_length.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: nodes.liquid
+section: nodeOperator
+date: Last Modified
+title: "Length Task"
+permalink: "docs/jobs/task-types/length/"
+---
+
+Returns the length of a byte array or string.
```

Comment:
Maybe we need to be clear on the behavior for unicode characters in a string. Does it return the number of runes in this case?